eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/io/ContigMongoReaderTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.ac.ebi.eva.accession.release.io;
```

Comment:
Add copyright and license header.

Reply:
can't believe I can't get intellij settings right ¬¬